frontend/src/metabase/entities/collections/collections.ts

```diff
@@ -58,6 +59,10 @@ const Collections = createEntity({
   displayNameMany: t`collections`,
 
   api: {
+    get: async (...params: any[]) => {
+      const response = await GET("/api/collection/:id")(params[0]);
+      return addTypesToCollections([response as unknown as Collection])[0];
+    },
```

Comment:
this is all destined for the fire when the backend is in, but this mutates individual collections too.

Reply:
🔥